x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_advanced_fields/index.tsx

```diff
@@ -405,7 +405,7 @@ export const AgentPolicyAdvancedOptionsContent: React.FunctionComponent<Props> =
                 namespace: (selectedOptions.length ? selectedOptions[0] : '') as string,
               });
             }}
-            isInvalid={Boolean(touchedFields.namespace && validation.namespace)}
```

Comment:
@elastic/fleet, please confirm that the value copied from `EuiFormRow` is relevant